src/thor/triplegbuilder.cc

```diff
@@ -864,6 +867,8 @@ TripLeg_Edge* AddTripEdge(const AttributesController& controller,
     }
   }
 
+  valhalla::thor::addIncidents(*trip_edge, graphreader, edge);
```

Comment:
Should we put this behind a request parameter?

Reply:
Yup, what about `incidents` which is similar to the existing `guidance_views`?

Reply:
Sounds good 👍

Reply:
i think this, like all the other things should be an attribute filter. check https://github.com/valhalla/valhalla/blob/master/valhalla/thor/attributes_controller.h for more information :smile:

Reply:
Ok, pushed a commit that makes use of the attributes-controller instead of a new proto option